packages/monorepo-tools/rewrite_history_from.sh

```diff
@@ -10,8 +10,25 @@
 # Example: rewrite_history_from.sh main-repository --branches
 
 SUBDIRECTORY=$1
-REV_LIST_PARAMS=${@:2}
+REV_LIST_PARAMS=`echo ${@:2} | tr " " "\n"`
 echo "Rewriting history from a subdirectory '$SUBDIRECTORY'"
+
+# Get directory of the other scripts
+MONOREPO_SCRIPT_DIR=$(dirname "$0")
+# Backup tag refs because `git filter-branch` doesn't backup tags
+$MONOREPO_SCRIPT_DIR/tag_refs_backup.sh
+
+# Remove tags that don't contain any files in $SUBDIRECTORY
```

Comment:
Im am kind of dumb user Svatopluk. So I like clear and describing comments. This comment means that tag can contain some files?

Reply:
contains means affected